pylint_plugins.py

```diff
@@ -255,3 +259,79 @@ def fix_ipa_classes(cls):
         fake_class(cls, ipa_class_members[class_name_with_module])
 
 MANAGER.register_transform(scoped_nodes.Class, fix_ipa_classes)
+
+
+class IPAChecker(BaseChecker):
+    __implements__ = IAstroidChecker
+
+    name = 'ipa'
+    msgs = {
+        'E9901': (
+            'Forbidden import %s',
```

Comment:
Maybe would be nice to write whihc rule has been applied `Forbidden import %s (%s should not import %s)`